docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,295 @@
+@weave.op()
+def ask_llama(question: str) -> str:
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": question}
+        ],
+    )
+    return response.choices[0].message.content
+
+# Call your function - Weave automatically traces everything
+result = ask_llama("What are the benefits of using W&B Weave for LLM development?")
+print(result)
+```
+
+## Step 2: Build a text summarization application
+
+Next, try running this code, which is a basic summarization app that shows how Weave traces nested operations:
+
+```python
+import weave
+import openai
+
+# Initialize Weave - replace with your-team/your-project
+weave.init("my-first-weave-project")
+
+@weave.op()
+def extract_key_points(text: str) -> list[str]:
+    """Extract key points from a text."""
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "Extract 3-5 key points from the text. Return each point on a new line."},
+            {"role": "user", "content": text}
+        ],
+    )
+    return response.choices[0].message.content.strip().split('\n')
+
+@weave.op()
+def create_summary(key_points: list[str]) -> str:
+    """Create a concise summary based on key points."""
+    points_text = "\n".join(f"- {point}" for point in key_points)
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "Create a one-sentence summary based on these key points."},
+            {"role": "user", "content": f"Key points:\n{points_text}"}
+        ],
+    )
+    return response.choices[0].message.content
+
+@weave.op()
+def summarize_text(text: str) -> dict:
+    """Main summarization pipeline."""
+    key_points = extract_key_points(text)
+    summary = create_summary(key_points)
+    return {
+        "key_points": key_points,
+        "summary": summary
+    }
+
+# Try it with sample text
+sample_text = """
+The Apollo 11 mission was a historic spaceflight that landed the first humans on the Moon 
+on July 20, 1969. Commander Neil Armstrong and lunar module pilot Buzz Aldrin descended 
+to the lunar surface while Michael Collins remained in orbit. Armstrong became the first 
+person to step onto the Moon, followed by Aldrin 19 minutes later. They spent about 
+two and a quarter hours together outside the spacecraft, collecting samples and taking photographs.
+"""
+
+result = summarize_text(sample_text)
+print("Key Points:", result["key_points"])
+print("\nSummary:", result["summary"])
+```
+
+## Step 3: Compare multiple models
+
+W&B Inference provides access to multiple models. Compare their performance:
+
+```python
+# Define a Model class to compare different LLMs
+class InferenceModel(weave.Model):
```

Comment:
As with above, I'd add the imports etc to make this standalone, and indicate below that example #4 builds on it.